include/dynd/type_sequence.hpp

```diff
@@ -321,7 +318,7 @@ struct outer;
 
 // Work around bug in MSVC 2015 update 1 and earlier.
```

Comment:
Fixed in https://github.com/libdynd/libdynd/commit/2c7fae5b5621dc0227326c6c31b8476fb03d0914.